### ISSUE TITLE
CAPTCHA: Keyboard focus indicator is not visible after error message

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#169]
**Page:** All (CAPTCHA under Report a problem or mistake on this page)
**URL:** https://recalls-rappels.canada.ca/en
**User Agent:** Firefox (v99)
**Language:** English and French

**Issue Description:** After the error message is generated, the checkbox is updated and becomes unchecked. At this point, when tabbing through, keyboard focus indicator is not visible for the checkbox. See screenshot below.

![image](https://user-images.githubusercontent.com/12256067/162785334-ca9dac21-5c43-4b78-96da-57d2b4bd4016.png)

**Steps to reproduce:**

1. Go to the above URL
2. Navigate the page using the "Tab" key
3. Access the Report a problem or mistake on this page component
4. Select the checkbox under CAPTCHA
5. Wait for the error message to appear
6. Continue tabbing

**Expected results:** The keyboard focus indicator should always be visible for the checkbox.

**Comment by echouin** [comment]
No longer an issue. resolved

**Comment by YuxuanXie01** [comment]
Retested, issue persists. Checkbox still lacks a focus indicator after the verification challenge expires.
![image](https://github.com/user-attachments/assets/c2d81340-d6db-4b41-aeca-dc619d0b5934)

**Comment by 40999221** [comment]
The issue still exist. After verification challenge expires, no focus on checkbox.
![image](https://github.com/user-attachments/assets/a959568f-775a-4c72-b233-01d3eb676ea9)

**Comment by zimw7** [comment]
Issue still exists. The keyboard focus indicator is not visible on the checkbox.
![image](https://github.com/user-attachments/assets/b4d36660-7cbf-4b92-a5c1-74e1c72d7c86)

**Comment by mmiron-HC** [comment]
@YuxuanXie01 Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605) Put the status of this one to Escalated and please include a note to say that this is a google product limitation. Add product_limitation to the labels as well
